nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -53,9 +53,9 @@ RQ_QUEUES = {
 # For detailed configuration see: https://github.com/Suor/django-cacheops#setup
 # Set Cache Ops variables
 redis_protocol = "rediss" if is_truthy(os.getenv("NAUTOBOT_REDIS_SSL", False)) else "redis"
-cache_ops_pwd = os.getenv("NAUTOBOT_REDIS_PASSWORD")
+cache_ops_pwd = os.getenv("NAUTOBOT_REDIS_PASSWORD", "")
 cache_ops_host = os.getenv("NAUTOBOT_REDIS_HOST", "localhost")
-cache_ops_user = os.getenv("NAUTOBOT_REDIS_USER")
+cache_ops_user = os.getenv("NAUTOBOT_REDIS_USER", "")
 cache_ops_port = int(os.getenv("NAUTOBOT_REDIS_PORT", 6379))
 
 CACHEOPS_REDIS = os.getenv("NAUTOBOT_CACHEOPS_REDIS", f"{redis_protocol}://:{cache_ops_pwd}@{cache_ops_host}:{cache_ops_port}/1")
```

Comment:
Should this be made conditional on `cache_ops_pwd`? It seems odd to have `redis://:@host...` in the case where no password is set -- wouldn't `redis://host...` be more correct?

Also we may need to percent-encode special characters in the password, similar to #425?